subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -598,12 +598,29 @@ config BT_CTLR_DYNAMIC_INTERRUPTS
 	  permit use of SoC's peripheral for custom use when Bluetooth is not
 	  enabled.
 
+choice
+	prompt "Optimazation options"
```

Comment:
Looks like a typo snuck in here: "Optimazation" -> "Optimization"

Reply:
Thanks, just realized that moving to `choice` means I need to also have the optimize for space which would be the alternative when optimize for speed need to be disabled (and no LTO) for the Controller.